packages/webkitgtk_6.rb

```diff
@@ -0,0 +1,179 @@
+      # index 77ebb802ebb03450b5e96629a47b6819a68672c6..d49d6e43d7eeb6673c624e00eadf3edfca0674eb 100644
+      #--- a/Source/cmake/WebKitCompilerFlags.cmake
+      #+++ b/Source/cmake/WebKitCompilerFlags.cmake
+      # @@ -143,6 +143,13 @@ if (COMPILER_IS_GCC_OR_CLANG)
+      # WEBKIT_PREPEND_GLOBAL_CXX_FLAGS(-Wno-nonnull)
+      # endif ()
+
+      #+    # This triggers warnings in wtf/Packed.h, a header that is included in many places. It does not
+      #+    # respect ignore warning pragmas and we cannot easily suppress it for all affected files.
+      #+    # https://bugs.webkit.org/show_bug.cgi?id=226557
+      #+    if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND ${CMAKE_CXX_COMPILER_VERSION} VERSION_GREATER_EQUAL "11.0")
+      #+        WEBKIT_PREPEND_GLOBAL_CXX_FLAGS(-Wno-stringop-overread)
+      #+    endif ()
+      #+
+      ## -Wexpansion-to-defined produces false positives with GCC but not Clang
+      ## https://bugs.webkit.org/show_bug.cgi?id=167643#c13
+      # if (CMAKE_CXX_COMPILER_ID MATCHES "GNU")
+      # GCCEOF
+      # File.write('gcc.patch', @gcc_patch)
+      # system 'patch -Np1 -F 10 -i gcc.patch'
+      # Patch from https://github.com/WebKit/WebKit/pull/1233
+      # downloader 'https://patch-diff.githubusercontent.com/raw/WebKit/WebKit/pull/1233.diff',
+      #           '70c990ced72c5551b01c9d7c72da7900d609d0f7891e7b99ab132ac1b4aa33ea'
+      # system "sed -i 's,data.pixels->bytes(),data.pixels->data(),' 1233.diff"
+      # system 'patch -Np1 -F 10 -i 1233.diff'
+      # Patch from https://github.com/WebKit/WebKit/pull/2926
+      # downloader 'https://patch-diff.githubusercontent.com/raw/WebKit/WebKit/pull/2926.diff',
+      # '26a8d5a9dd9d61865645158681b766e13cf05b3ed07f30bebb79ff73259d0664'
+      # system "sed -i '22,63d' 2926.diff"
+      # system 'patch -Np1 -F 10 -i 2926.diff'
+      # @arch_flags = '-mtune=cortex-a15 -mfloat-abi=hard -mfpu=neon -mtls-dialect=gnu -marm -mlibarch=armv8-a+crc+simd -march=armv8-a+crc+simd'
+      @arch_flags = '-mfloat-abi=hard -mtls-dialect=gnu -mthumb -mfpu=vfpv3-d16 -mlibarch=armv7-a+fp -march=armv7-a+fp'
+    end
+    @gcc_ver = '-10' if %w[aarch64 armv7l x86_64].include? ARCH
+    @new_gcc = <<~NEW_GCCEOF
+      #!/bin/bash
+      gcc#{@gcc_ver} #{@arch_flags} $@
+    NEW_GCCEOF
+    @new_gpp = <<~NEW_GPPEOF
+      #!/bin/bash
+      g++#{@gcc_ver} #{@arch_flags} $@
+    NEW_GPPEOF
+    FileUtils.mkdir_p 'bin'
+    File.write('bin/gcc', @new_gcc)
+    FileUtils.chmod 0o755, 'bin/gcc'
+    File.write('bin/g++', @new_gpp)
+    FileUtils.chmod 0o755, 'bin/g++'
+  end
+
+  def self.build
+    # This builds webkit2gtk5 (which uses gtk4, but not libsoup2)
+    @workdir = `pwd`.chomp
+    # Bubblewrap sandbox breaks on epiphany with
+    # bwrap: Can't make symlink at /var/run: File exists
+    # LDFLAGS from debian: -Wl,--no-keep-memory
+    unless File.file?('build.ninja')
+      @arch_linker_flags = ARCH == 'x86_64' ? '' : '-Wl,--no-keep-memory'
```

Comment:
Conditional not needed?

Reply:
same reason as others... needed in case arm starts working...